test-data/assign.yaml

```
@@ -422,3 +437,6 @@
   - r2.svalue=[0, 4294967295]
   - r2.uvalue=[0, 4294967295]
   - r2.svalue=r2.uvalue
+
+messages:
+  - "0: Invalid type (r1.type == number)"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)